setup.py

```diff
@@ -0,0 +1,18 @@
+from setuptools import setup
+
+setup(
+    name="simpleocr",
+    packages=["simpleocr"],
+    version="1.0.0",
+    description="A library for simple OCR in Python using OpenCV",
+    author="The simple-ocr-opencv authors",
+    url="https://www.github.com/goncalopp/simple-ocr-opencv",
+    download_url="https://www.github.com/goncalopp/simple-ocr-opencv/archive/1.0.0.tar.gz",
+    keywords=["OCR", "OpenCV"],
+    license="AGPL",
+    classifiers=["Programming Language :: Python :: 2.7",
+                 "Programming Language :: Python :: 3",
+                 "License :: OSI Approved :: GNU Affero General Public License v2 or later (AGPLv2+)"],
+    include_package_data=True,
+    install_requires=["opencv-python"]
```

Comment:
That sounds logical and that help print should work. 

The average user, and most programmers just as well, indeed download untrusted binaries all the time: That's what anti-virus is for, right? :P Kidding aside, the alternative on Windows is downloading the OpenCV source code and compiling the OpenCV with `cmake` and an appropriate C-compiler, which is pretty difficult. It is also possible to use the Visual Studio compiler (I think, I'm not even sure on that), which would be easier, but requires non-free (in the worst form) third-party software to even get it up and running.

Of course, there are those few who can actually compile OpenCV and install it. Perhaps it would be even possible to create a script that not only prints the options, but also provides the user with the option to install `opencv-python` through this snippet:
```python
import pip

pip.main("install", "opencv-python")
```
This would keep things easy, but also allow the user freedom in the choice of how `opencv-python` must be installed, on all platforms.